test/Microsoft.AspNet.Mvc.Testing.Test/ControllerTestHelperTest.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
@rynowak 2 examples that you pointed out are really full-blown controllers. For now, I'm going to use Mock as much as possible for the all external components such as EF, UserManager, and so on. Plus, those tests are not going to be checked in the repository, but they are just to show how users codes look like..still I think it will takes some time..